src/scripts/main.js

```diff
@@ -18,10 +18,29 @@ const food = {
   },
 };
 
+const treeDiv = document.createElement('div');
+
+treeDiv.setAttribute('id', 'tree');
+
+document.querySelector('body').insertAdjacentElement('afterbegin', treeDiv);
+
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
   // WRITE YOUR CODE HERE
+  for (const key in data) {
+    const tempUl = document.createElement('ul');
+    const tempLi = document.createElement('li');
+
+    tempLi.textContent = key;
+    tempUl.appendChild(tempLi);
+
+    element.appendChild(tempUl);
+
+    if (data[key].length !== 0) {
```

Comment:
The condition `data[key].length !== 0` is incorrect for checking if the nested data is empty. Since `data[key]` is an object, it does not have a `length` property. You should check if the object has any keys using `Object.keys(data[key]).length !== 0` or simply `Object.keys(data[key]).length > 0`.